tests/server/api/test_collections.py

```diff
@@ -57,7 +66,7 @@ def mock_get_view(
             return_value=Response(200, json=mock_block_response)
         )
         respx_mock.get(self.collection_view_url("worker")).mock(
-            return_value=Response(404, json=mock_collection_response)
+            return_value=Response(200, json=mock_worker_response)
```

Comment:
why is this a `200` now?

Reply:
That mock wasn't being used, so I repurposed it.

Reply:
gotcha